Sources/Components/Label/Label.swift

```diff
@@ -44,11 +48,28 @@ public class Label: UILabel {
         isAccessibilityElement = true
 
         accessibilityLabel = text
-        textColor = style?.color
         font = style?.font
+
+        if let color = color(for: style) {
+            textColor = color
+        }
     }
 
     // MARK: - Dependency injection
 
     public var style: Style?
+
+    // Required for backwards compatibility https://github.com/finn-no/FinniversKit/pull/217
+    private func color(for style: Style?) -> UIColor? {
```

Comment:
Instead of keeping this for backwards compatibility, how about setting licorice as default color? We should normally never use labels with nil/black textColor anyway, right?

Reply:
Good point, I'll change it :D